.github/workflows/build.yml

```diff
@@ -63,10 +66,13 @@ jobs:
         name: Build in Docker (x86_64)
         run: make wheel-manylinux IMAGE=quay.io/pypa/"$MANYLINUX_VERSION"_x86_64
 
-      - uses: actions/upload-artifact@v3.1.1
+      - uses: actions/upload-artifact@v4
         with:
-          name: ${{ github.sha }}
+          name: artifact-linux-${{ matrix.build-arch }}
```

Comment:
Not sure if the name matters here. `matrix.target` seems to be unavailable so I used `matrix.build-arch` instead.

Reply:
I'm not sure either